### PR TITLE
Phase 1: Gmail Support Package for Ushahidi Platform

### PR DESCRIPTION
**Pull request by webong**

### CODE REVIEW

**Comment by webong** [comment]
Hello @rowasc @tuxpiper ... sorry, I forgot to inform you about this PR, I have updated it with my latest work.

File: src/GmailServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+     * @return void
+     */
+    private function registerGmailSource()
+    {
+        if (! $this->shouldRegisterGmailSourceDriver()) {
+            return;
+        }
+
+        $this->app['datasources']->extend('gmail', function ($config) {
+            $config['user'] = 'wisdomaebong@gmail.com';
```
Comment (webong) [inline src/GmailServiceProvider.php]: This is just for test purpose, the $config['user'] will be provided by the config repository.

**Comment by tuxpiper** [comment]
Alright :+1: thank you @webong  !

**Comment by tuxpiper** [comment]
@webong  I'll be checking this before we talk again .  Would it be possible to add in a README file that explains a bit what we have here in this repo and how to use it?

**Comment by webong** [comment]
Okay, I'll make a ReadMe.MD file

**Comment by tuxpiper** [comment]
Looking pretty good to me :+1: I think you could merge this @webong  and then we continue from there

**Pull request merged**